### DISCUSSION TITLE
Billing code type version - Out-Of-Network Allowed Amount File

### DISCUSSION THREAD

**Discussion by Brinkley100** [CMSgov/price-transparency-guide#167] [Q&A] [answered]
When billing_code is NDC what do we populate in the  billing_code_type_version field?

**Comment by bryanbriegel** [comment]
Good question - the schema notes "There might be versions associated with the billing_code_type. For example, Medicare's current (as of 5/24/21) MS-DRG version is 37.2" - and Required "Yes" -  my read would be this field should be "situational"

**Comment by tdfow** [comment]
Based on current schema indicating required, when there isn't a specific version like DRGs, we have been using the CCYY as the version to represent the current year the billing code value is 'valid'.

**Reply by Brinkley100** [reply]
That is a good idea. Thank you.

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping up an old thread -- documentation on how to handle coding sets that do not have a version can [be found here](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#in-network-object). 

TL;DR - If there is no version available for the billing_code_type, use the current plan's year YYYY that is being disclosed.
